plugins/modules/rds_cluster.py

```diff
@@ -389,6 +405,23 @@
     username: "{{ username }}"
     cluster_id: "cluster-{{ resource_prefix }}-restored"
     snapshot_identifier: "cluster-{{ resource_prefix }}-snapshot"
+
+- name: Create an Aurora PostgreSQL cluster and attach an intance
+  rds_cluster:
+    state: present
+    engine: aurora-postgresql
+    engine_mode: provisioned
+    cluster_id: '{{ cluster_id }}'
+    username: '{{ username }}'
+    password: '{{ password }}'
+
+- name: Attach a new instance to the cluster
+  rds_instance:
```

Comment:
```suggestion
  amazon.aws. rds_cluster:
```

Reply:
Sorry, I did the change ^^ but I believe I got confused by the fact you own this branch and I ended up doing the push at the wrong location.